src/components/CartItemList/calculate.ts

```diff
@@ -0,0 +1,13 @@
+import { CartItemType } from '../../types/cartItem';
+
+export const calculateOrderAmount = (items: CartItemType[]): number => {
+  return items.reduce((acc, item) => acc + item.product.price * item.quantity, 0);
+};
+
+export const calculateDeliveryFee = (orderAmount: number): number => {
+  return orderAmount > 100000 ? 0 : 3000;
```

Comment:
큰 숫자를 사용할 때는 numeric sperator를 같이 활용하시면 좋을거 같네요
```suggestion
  return orderAmount > 100_000 ? 0 : 3_000;
```

Reply:
처음 알게된 표기법이네요! 감사합니다!